src/helpers/mailer.js

```diff
@@ -9,6 +9,33 @@ const templates = {
   password_reset: 'd-3fa45363fe634b418146f72cbe03942b',
   confirm_account: 'd-84376bbc49e24647ab77848bcb926eb5'
 };
+export const sendEmail = {
+  /**
+   * Email Transporter
+   * @method sender
+   * @memberof Mailer
+   * @param {string} email
+   * @param {string} subject
+   * @param {string} message
+   * @returns {nothing} returns nothing
+   */
+  sender(data) {
```

Comment:
I do not think we need this template, there is one already like this in this same module on `develop`. Please use that https://github.com/andela/ah-kifaru-backend/blob/develop/src/helpers/mailer.js